### DISCUSSION TITLE
Welcome to Discussions!

### DISCUSSION THREAD

**Discussion by SebConejo** [mnfst/llm-gateway#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by BlaqueC88** [comment]
Hi, I recently discovered Case and would like to try it but as a new dev, I'm not sure how to use it with Vite-React and/or Next.js. Is there documentation for these?

**Reply by SebConejo** [reply]
Hello @BlaqueC88 , We have created quick starts which are available here: 
- https://docs.case.app/react
- https://docs.case.app/vue

Does this help?

For your information, following the release of CASE, we had a lot of positive and interesting feedback. This feedback has enabled us to work on a new product that will replace CASE. This new product is an improvement on what we're trying to offer with CASE.  It will be a product with a clearer promise. If you'd like to be kept informed as soon as it's released, 
1. [subscribe to our newsletter](https://docs.google.com/forms/d/1fCfkSELKgzcjlVcoaZwIXHzgcBsArEuf7x9imUFEBU4/edit) 
2. join us on our [Discord](https://discord.gg/FepAked3W7).
